packages/common-db/src/lib/common/objects.ts

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+import * as fs from 'fs-extra';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Please no :) Prefer `// @ts-expect-error Explanation why this must be ignored` over `ts-ignore`.
This has the advantage of becoming an error when it is no longer needed, i.e. when we do have typings.

Also see my comment regarding ESLint rules (editing ban-ts-comment to allow ts-expect-error).